item.py

```diff
@@ -249,7 +253,10 @@ def parseItem(m, compendium, args):
 
     if 'strength' in m:
         strength = ET.SubElement(itm, 'strength')
-        strength.text = str(m['strength'])
+        if m['strength']:
+            strength.text = str(m['strength'])
+        else:
+            strength.text = ""
```

Comment:
This block will never be executed.

Reply:
Some items have `"strength": null`, I think those were hitting the `else`

Reply:
Yeah, I understand, but it should be fixed like this, otherwise the else wont get hit:
```suggestion
    if 'strength' in m and m['strength']:
         strength = ET.SubElement(itm, 'strength')
         strength.text = str(m['strength'])
    else:
         strength.text = ""
```

Reply:
Is there a better way to handle that?

Reply:
I'll change to `strength.text = str(m['strength'] or '')`

Reply:
Thats better